Could not find "element-connector/0.0.0" in the repositories of the current location

Using the last source code, building with maven leads to this error:
```
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/0.9.0/Java/ (1,18kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/0.9.0/Java/ (1,18kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.0/Java/ (1,17kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.0/Java/ (1,17kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.1/Java/ (1,18kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.1/Java/ (1,18kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.2/Java/ (1,18kB)
[INFO] Fetching content.jar from http://download.eclipse.org/paho/releases/1.0.2/Java/ (1,18kB)
[INFO] Adding repository http://eclipse.github.io/smarthome/third-party/target/repository
[INFO] Fetching content.jar from http://eclipse.github.io/smarthome/third-party/target/repository/ (5,06kB)
[INFO] Fetching content.jar from http://eclipse.github.io/smarthome/third-party/target/repository/ (5,06kB)
[ERROR] Failed to resolve target definition D:\dev\openhab2-master\git\smarthome\targetplatform\smarthome.target: Could not find "element-connector/0.0.0" in the repositories of the current location
```

--- a/targetplatform/smarthome.target
+++ b/targetplatform/smarthome.target
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?pde version="3.8"?><target name="SmartHome" sequenceNumber="110">
+<?pde version="3.8"?><target name="SmartHome" sequenceNumber="112">
 <locations>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="com.google.inject" version="3.0.0.v201312141243"/>
@@ -150,8 +150,8 @@
 <unit id="org.osgi.service.jdbc" version="0.0.0"/>
 <unit id="org.osgi.service.jdbc.source" version="0.0.0"/>
 <unit id="californium-osgi" version="0.0.0"/>
-<unit id="element-connector" version="0.0.0"/>
-<unit id="scandium" version="0.0.0"/>
+<unit id="org.eclipse.californium.element-connector" version="0.0.0"/>
+<unit id="org.eclipse.californium.scandium" version="0.0.0"/>
 <repository location="http://eclipse.github.io/smarthome/third-party/target/repository"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">

